packages/aws_common/lib/src/util/stoppable_timer.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'dart:async';
+
+import 'package:meta/meta.dart';
+
+/// {@template amplify_analytics_pinpoint_dart.stoppable_timer}
+/// A Timer that can be stopped and started again.
+/// {@endtemplate}
+@protected
+class StoppableTimer {
```

Comment:
Question: Can StoppableTimer be removed from amplify_analytics_pinpoint?

Reply:
Good catch. I initially moved StoppableTimer into amplify_core, but forgot to remove it after migrating it again to aws_common.